src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -66,8 +66,8 @@ class DistributionConfiguration extends Component {
                 this.loading = false;
                 const { job } = nextProps;
                 if (job && job.fieldModels) {
-                    const channelModel = nextProps.job.fieldModels.find(model => model.descriptorName.startsWith('channel_'));
-                    const providerModel = nextProps.job.fieldModels.find(model => model.descriptorName.startsWith('provider_'));
+                    const channelModel = nextProps.job.fieldModels.find(model => FieldModelUtilities.hasKey(model, KEY_NAME));
```

Comment:
Does KEY_NAME need to be imported?

Reply:
It does not because it is exported as a const in this javascript file.